### ISSUE TITLE
"Auto is not initialized! Make sure the have run Auto.loadConfig" after adding magic-zero plugin

### ISSUE THREAD

**Issue by hasparus** [intuit/auto#1946]
**Describe the bug**

I'm getting the following error
```
Error: Auto is not initialized! Make sure the have run Auto.loadConfig
```

The error started occuring when I added `magic-zero` plugin, after removing the plugin the problem goes away. 

**To Reproduce**

You can go here to take a look at the log: https://github.com/system-ui/theme-ui/runs/2364762509

If you want to reproduce it, grab [my config](https://github.com/system-ui/theme-ui/blob/develop/auto.config.ts) and run `auto info` 

**Expected behavior**

magic-zero doesn't crash :c

**Environment information:**

After commenting out magic-zero:

```txt
Environment Information:

"auto" version: v10.25.0
"git"  version: v2.27.0
"node" version: v14.15.3

Project Information:

✔ Repository:      system-ui/theme-ui
✔ Author Name:     hasparus
✔ Author Email:    hasparus@gmail.com
✔ Current Version: v0.7.0
✔ Latest Release:  v0.7.0

✔ Labels configured on GitHub project 

GitHub Token Information:

✔ Token:            [Token starting with 6cf3]
✔ Repo Permission:  admin
✔ User:             hasparus
✔ API:              
✔ Enabled Scopes:   admin:org, admin:org_hook, admin:public_key, admin:repo_hook, gist, notifications, repo, user, workflow
✔ Rate Limit:       4948/5000
```

**Comment by hydrosquall** [comment]
I ran into this problem as well, and can reproduce with a local `auto info`. 

```bash
Error: Auto is not initialized! Make sure the have run Auto.loadConfig
    at Auto.createErrorMessage (/PROJECT/node_modules/@auto-it/core/src/auto.ts:2041:12)
    at Auto.prefixRelease (/PROJECT/node_modules/@auto-it/core/src/auto.ts:2031:18)
    at getPreviousVersion (/PROJECT/node_modules/@auto-it/npm/src/index.ts:296:16)
```

Logfile [link](https://github.com/vega/ts-json-schema-generator/pull/856/checks?check_run_id=3094732626)

It can be reproduced with a 2-plugin config. 

```ts
export default function rc(): AutoRc {
    return {
        plugins: [
            "npm",
            "magic-zero"
        ]
}
```

I suspect there's a bug in the interaction between the `npm` and the `magic-zero` plugin. `magic-zero` together with `conventional-commits` did not create any problems. 

I'm not familiar enough with the auto lifecycle to debug this, but hopefully this breadcrumb trail can help a maintainer. I also tried copying over the `index.ts` file to fork the plugin for debugging, but ran into several TS compiler errors.

For now, we'll probably drop usage of the `magic-zero` plugin too. https://github.com/vega/ts-json-schema-generator/pull/856/
